### DISCUSSION TITLE
How to set preferredcolorscheme

### DISCUSSION THREAD

**Discussion by Daniellled** [MicrosoftEdge/WebView2Feedback#4115] [Q&A]
I am using the WebView2 control for WPF. I see API's related to setting the browser theme, dark, light, OS default. - https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2profile?view=webview2-1.0.1587.40 and https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2profile.preferredcolorscheme?view=webview2-dotnet-1.0.1189-prerelease#microsoft-web-webview2-core-corewebview2profile-preferredcolorscheme

Where do I set that profile? I am simply using the WebView2 control in my xaml and invoking MyWebView2.EnsureCoreWebView2Async(); in the code behind.

Is it possible to set this property via dependency property. Similar to how you can set the DefaultBackgroundColor?

**Comment by victorhuangwq** [comment]
@peiche-jessica would you be able to help this user out?

**Reply by peiche-ms** [reply]
@johna-ms Would you be able to help about this please?

**Comment by johna-ms** [comment]
@Daniellled you can use the [PreferredColorScheme](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2profile.preferredcolorscheme?view=webview2-dotnet-1.0.2088.41#microsoft-web-webview2-core-corewebview2profile-preferredcolorscheme) property to set this value. It is on the [CoreWebView2.CoreWebView2Profile](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.profile?view=webview2-dotnet-1.0.2088.41#microsoft-web-webview2-core-corewebview2-profile) object Hope this helps!
